docs/content/en/docs-dev/feature-status/_index.md

```diff
@@ -118,6 +118,7 @@ Please note that the phases (Incubating, Alpha, Beta, and Stable) are applied to
 | Support GCP [GCS](https://cloud.google.com/storage) as file store | Beta |
 | Support AWS [S3](https://aws.amazon.com/s3/) as file store | Beta |
 | Support [Minio](https://github.com/minio/minio) as file store | Beta |
+| Support using file storage as database | Incubating |
```

Comment:
🤔  How about making it to be more clearer like this?

`Support using file storage such as GCS, S3, Minio for both data store and file store (It means no database is required to run control plane)`

Reply:
We could add a link to the docs that explains more about this feature in detail when we create documentation for it, but you're right. In the meantime, a clear note is better 👍 Let me adopt your words.

Reply:
Thank you.

Reply:
Thanks, addressed by https://github.com/pipe-cd/pipe/pull/2983/commits/8e5010315f19a56afe2ded072b0f8669549e3b4d 🙏